autotest-agent/src/jvmMain/kotlin/com/epam/drill/test/agent/transport/AdminMessageSender.kt

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.test.agent.transport
+
+import java.io.File
+import io.aesy.datasize.ByteUnit
+import io.aesy.datasize.DataSize
+import mu.KotlinLogging
+import com.epam.drill.agent.configuration.DefaultParameterDefinitions
+import com.epam.drill.agent.transport.InMemoryAgentMessageQueue
+import com.epam.drill.agent.transport.JsonAgentMessageSerializer
+import com.epam.drill.agent.transport.QueuedAgentMessageSender
+import com.epam.drill.agent.transport.RetryingTransportStateNotifier
+import com.epam.drill.agent.transport.http.HttpAgentMessageDestinationMapper
+import com.epam.drill.agent.transport.http.HttpAgentMessageTransport
+import com.epam.drill.common.agent.transport.AgentMessageDestination
+import com.epam.drill.common.agent.transport.AgentMessageSender
+import com.epam.drill.plugins.test2code.api.Action
+import com.epam.drill.test.agent.configuration.Configuration
+import com.epam.drill.test.agent.configuration.ParameterDefinitions
+
+object AdminMessageSender : AgentMessageSender<Action> {
+
+    private const val QUEUE_DEFAULT_SIZE: Long = 512L * 1024 * 1024
+
+    private val logger = KotlinLogging.logger {}
+    private val messageSender = messageSender()
+
+    override val available: Boolean
+        get() = messageSender.available
+
+    override fun send(destination: AgentMessageDestination, message: Action) =
+        messageSender.send(destination, message)
+
+    private fun messageSender(): QueuedAgentMessageSender<Action, ByteArray> {
+        val transport = HttpAgentMessageTransport(
+            Configuration.parameters[ParameterDefinitions.ADMIN_ADDRESS],
+            Configuration.parameters[ParameterDefinitions.API_KEY],
+            Configuration.parameters[ParameterDefinitions.SSL_TRUSTSTORE].let(::resolvePath),

```

Comment:
do we have an option to skip setting up `ssl_trusstore` and allow to ignore validity of ssl sertificate of accepting side?

Reply:
if we dont pass SSL_TRUSTORE it will work exactly that way